stats/DataAggregation.md

```diff
@@ -0,0 +1,71 @@
+# Data Aggregation API Overview
+The stats library allows users to define views to configure how data are aggregated and broken down.
+The core data types used are:
+* `Aggregation`: defines the aggregation type (e.g. `Distribution`, `Sum`, `Mean`).
+* `View`: defines how individual measurements are broken down by tags (similar to labels in some 
+other systems) and aggregated.
+
+## Aggregation
+An Aggregation describes how data collected is aggregated.
+
+The library SHOULD provide support for multiple types of Aggregations:
+* `Count`: counts the number of measurements recorded.
+* `Sum`: indicates that data collected and aggregated with this `Aggregation` will be summed up.
+* `Mean`: indicates that data collected and aggregated with this `Aggregation` will calculate the
```

Comment:
I worry about this one. `Mean` isn't aggregable across dimensions, so doesn't seem particularly useful in general. For distribution summary type metrics, shipping a total amount and count is generally sufficient to be able to derive a truly aggregable mean. Some systems are so aware of this problem that they provide a standard query function OOTB (see [dist-avg](https://github.com/Netflix/atlas/wiki/math-dist%E2%80%90avg). Some monitoring systems expect mean to be pre-computed as a matter of convention, but could shipping mean be an exporter concern for those systems?

Reply:
easy calculate with Prometheus too.

Reply:
Drop this for the moment. Added an issue to discuss the need for this https://github.com/census-instrumentation/opencensus-specs/issues/56